account_reconcile_analytic_tag/models/account_reconcile_model.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class AccountReconcileModel(models.Model):
+    _inherit = "account.reconcile.model"
+
+    # TODO: Remove if merging https://github.com/odoo/odoo/pull/188808
+    def _get_write_off_move_lines_dict(self, residual_balance, partner_id):
+        res = super()._get_write_off_move_lines_dict(
+            residual_balance=residual_balance, partner_id=partner_id
+        )
+        if len(res) == 0:
+            return res
+        key = 0
```

Comment:
For usual nomenclature (`key` is used in dictionaries), this should be:

```suggestion
        index = 0
```

Anyway, look for the next suggestion for saving some lines.